Whatsapp_Chat_Exporter/__main__.py

```diff
@@ -429,14 +483,18 @@ def main():
                 if args.android:
                     android_handler.calls(db, data, args.timezone_offset, filter_chat)
             if not args.no_html:
+                if contact_store.is_empty():
```

Comment:
```suggestion
                if not contact_store.is_empty():
```

Reply:
Ah. This should be fixed in the definition.